docs/direct-access-urls.md

```diff
@@ -96,14 +101,21 @@ _caching content store_), should have dedicated configuration options:
     - If not set, the default system-wide setting will be used
       (`system.directAccessUrl.maxExpiryTimeInSec`).
 
+>**Note:** `{$connectorName}` depends on Alfresco Cloud Connector. For S3 Connector it is 's3', for Azure Connector it is 'az'.
+
 >**Note:** Callers within the platform (i.e. Java interfaces) can either request a specific
 > expiry time or rely on the default.
```

Comment:
Minor: Do we clear state the OOTB (out-of-the-box) default values ? 

For example, what is the setting for:

- system.directAccessUrl.defaultExpiryTimeInSec ?
- connector.{$connectorName}.directAccessUrl.defaultExpiryTimeInSec (is it 30 secs as above for both s3 and az) ?


Either add here (or link to config code) ?

Reply:
In the sections above there are proposed default values for all of these properties (30 seconds). I added a reminder about that here.
I think that adding links to config code may make some mess in the future when the code gets changed

Reply:
Thanks. 

I might have missed it but I couldn't see the default (OOTB) value for 'system.directAccessUrl.maxExpiryTimeInSec'.